internal/ui/ui_js.go

```diff
@@ -711,8 +712,21 @@ func (u *UserInterface) appendDroppedFiles(data js.Value) {
 		return
 	}
 
-	fs := items.Index(0).Call("webkitGetAsEntry").Get("filesystem").Get("root")
-	u.inputState.DroppedFiles = file.NewFileEntryFS(fs)
+	var files []*file.FileEntryFS
+	for i := 0; i < items.Length(); i++ {
+		kind := items.Index(i).Get("kind").String()
+		switch kind {
+		case "file":
+			fs := items.Index(i).Call("webkitGetAsEntry").Get("filesystem").Get("root")
+			files = append(files, file.NewFileEntryFS(fs))
```

Comment:
If we adopt only one 'file' object, what's the point of accumulating them? What about 

```go
var f *file.FileEntryFS
for ... {
    // ...
        f = file.NewFileEntryFS(fs)
        break
    // ...
}
if f == nil {
    return
}
```

Reply:
Sounds good. This would also highlight that only the first file is processed since we're using an early break.